akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -1051,8 +1051,7 @@ trait DefaultSSLContextCreation {
         WarningAboutGlobalLoose)
 
     if (sslConfig.config.loose.disableSNI) {
-      log.warning("Detected that Server Name Indication (SNI) is disabled globally (via ssl-config's akka.ssl-config.loose.disableSNI) for the Http extension! " +
-        WarningAboutGlobalLoose)
+      log.warning("Detected that Server Name Indication (SNI) is disabled globally (via ssl-config's akka.ssl-config.loose.disableSNI) for the Http extension!")
```

Comment:
Removed the drastic part of that warning. Might still be good to warn if it globally disabled, as it may prevent accessing multi-homed servers.